workspaces/azure-devops/plugins/scaffolder-backend-module-azure-devops/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@backstage-community/plugin-scaffolder-backend-module-azure-devops",
+  "description": "The azure-devops module for @backstage/plugin-scaffolder-backend",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
You need to remove this line as with this in place it won't get published to NPM